internal/policy/verify.go

```diff
@@ -445,70 +451,6 @@ func VerifyCommit(ctx context.Context, repo *git.Repository, ids ...string) map[
 	return status
 }
 
-// VerifyTag verifies the signature on the RSL entries for the specified tags.
```

Comment:
Let's scope PRs. Can we address VerifyCommit/VerifyTag in separate PRs? This file is possibly the single most important in gittuf, so let's have small, easily reviewable changes. :smile: